code/game/objects/items/storage/pouch.dm

```diff
@@ -1240,6 +1240,14 @@
 		icon_state = "[initial(icon_state)]_full"
 	else
 		icon_state = initial(icon_state)
+		
+/obj/item/storage/pouch/machete/_item_insertion(obj/item/W, prevent_warning = 0)
+	..()
+	playsound(src, soundin = 'sound/weapons/gun_rifle_draw.ogg', vol = 15, vary = TRUE)
+
+/obj/item/storage/pouch/machete/_item_removal(obj/item/W, atom/new_location)
+	..()
+	playsound(src, soundin = 'sound/weapons/gun_rifle_draw.ogg', vol = 15, vary = TRUE)
```

Comment:
make these vars like
`	var/sheatheSound = 'sound/weapons/gun_pistol_sheathe.ogg'
	var/drawSound = 'sound/weapons/gun_pistol_draw.ogg'`
having them as proc stuff is bleh